src/commands/addons/delete.js

```diff
@@ -1,12 +1,11 @@
 const Command = require('@netlify/cli-utils')
 const { getAddons, deleteAddon } = require('netlify/src/addons')
-const { parseRawFlags } = require('../../utils/parse-raw-flags')
 const { flags } = require('@oclif/command')
 
 class AddonsDeleteCommand extends Command {
   async run() {
     const accessToken = await this.authenticate()
-    const { args, raw } = this.parse(AddonsDeleteCommand)
```

Comment:
raw flags are needed to parse all the variable options for add-on commands

I think its ok to remove for the delete command here but just a heads up.

We might need to add back in, if use case arises where add-ons accept args on deletions.

Reply:
I had to change this because the `delete` command was not respecting the `--force` flag for some reason. But i tried to keep the change to minimal.